monkey/monkey_island/cc/resources/reset_configured_propagation_credentials.py

```diff
@@ -0,0 +1,27 @@
+from flask import make_response
+
+from monkey_island.cc.repository import RemovalError
+from monkey_island.cc.repository.i_credentials_repository import ICredentialsRepository
+from monkey_island.cc.resources.AbstractResource import AbstractResource
+from monkey_island.cc.resources.request_authentication import jwt_required
+
+
+class ResetConfiguredPropagationCredentials(AbstractResource):
+    urls = ["/api/reset-configured-propagation-credentials"]
```

Comment:
I know we wrote a task to do this, but in looking at it, it looks wrong.

We can just post an empty list to the existing endpoint and remain RESTful.